pw-pbkdf2.c

```diff
@@ -229,6 +235,81 @@ static int pbkdf2_encrypt(
 						  dk.bv_len, iteration,
 						  salt.bv_len, (const uint8_t *) salt.bv_val,
 						  dk.bv_len, dk_value);
+#elif HAVE_MOZNSS
+	SECStatus rv;
+	rv = NSS_NoDB_Init(".");
+	if (rv != SECSuccess)
+	{
+		fprintf(stderr, "NSS initialization failed (err %d)\n",
+		PR_GetError());
+	}
+	PK11SlotInfo *slot = NULL;
+	SECAlgorithmID * algid = NULL;
+	PRArenaPool *arena_001;
+	PRArenaPool *arena_002;
+	PRArenaPool *arena_003;
+	arena_001 = PORT_NewArena(DER_DEFAULT_CHUNKSIZE);
```

Comment:
We must assume Insufficient memory.  NULL check.

Reply:
Done